src/main/java/uk/gov/hmcts/reform/iahearingsapi/domain/handlers/servicedatahandlers/HandlerUtils.java

```diff
@@ -64,7 +63,7 @@ public static LocalDateTime getHearingDateAndTime(LocalDateTime hearingDateTime,
                                                       List<HearingChannel> hearingChannels,
                                                       String venueId) {
         if (hearingChannels.contains(INTER)) {
-            return StringUtils.equals(venueId, GLASGOW_EPIMMS_ID)
```

Comment:
It was causing errors in my local and it is not necessary to use StringUtils for this.